### ISSUE TITLE
Plots in k25 with few RAM

### ISSUE THREAD

**Issue by Manzito13** [madMAx43v3r/chia-plotter#355]
Hello .. for those of us who have few resources and want to use RamDrive .. how can we plot in k25 format through the MAX code, using 2 temporary partitions in RAM and then depositing the final plots on an HDD? In the Official Client Chia I was able to plot in k25 using RamDrive but it would be very interesting to see how this works with MADMAX ... Thank you !!!

**Comment by fabitom** [comment]
Why you want to plot k25 ?

**Comment by cayolblake** [comment]
Interestingly weird request!

**Comment by Manzito13** [comment]
> 
> 
> Why you want to plot k25 ?

Yes .. as incredible as it may seem .. I must adapt to the resources that exist .. hehe .. I use 2 RamDrive partitions and in a term of 4:30 hours approx. I get the equivalent of a k32 plot (101.XXX KB). but I want to test this with the MADMAX system because with Chia Oficial I can only plot a maximum of 29 plots at a time ... and it is also surprising but cultivates the plots without problems

**Comment by VertiHydro** [comment]
> Hello .. for those of us who have few resources and want to use RamDrive .. how can we plot in k25 format through the MAX code, using 2 temporary partitions in RAM and then depositing the final plots on an HDD? In the Official Client Chia I was able to plot in k25 using RamDrive but it would be very interesting to see how this works with MADMAX ... Thank you !!!

But why do you want to plot k25s? You can't do anything with them - the minimum plot size for farming is k32!

**Comment by Manzito13** [comment]
> 
> 
> > Hello .. for those of us who have few resources and want to use RamDrive .. how can we plot in k25 format through the MAX code, using 2 temporary partitions in RAM and then depositing the final plots on an HDD? In the Official Client Chia I was able to plot in k25 using RamDrive but it would be very interesting to see how this works with MADMAX ... Thank you !!!
> 
> But why do you want to plot k25s? You can't do anything with them - the minimum plot size for farming is k32!
Yes, it's crazy right ??? .. but it's true .. it's incredible, it works !! .. the official client Chia launches a warning but still allows it .. 
![Incredible](https://user-images.githubusercontent.com/85766978/121793227-6c460c00-cbd3-11eb-8891-3a1f2c9a16b8.jpg)

**Comment by Manzito13** [comment]
and not only that, I have tried it on the command line in Powershell and it does not work .. because it does not allow the k25 argument and throws a message in cmd saying that the minimum plotting is K32 ... it really is very strange ... that's why It would be great to try the MADMAX method using the k25 parameter but I would not really know how to do it ...

**Comment by TipTaco** [comment]
Despite the GUI showing those k=25 plots as farming, they will _never_ win a reward on mainnet. Anything less than k=32 will be skipped over when proving.

**Comment by VertiHydro** [comment]
> Despite the GUI showing those k=25 plots as farming, they will _never_ win a reward on mainnet. Anything less than k=32 will be skipped over when proving.

That's what I thought - k25 is only for testnet and has no function on mainnet. @Manzito13 I suggest you delete those plots and begin plotting some k32s because you can't win anything with a k25.

**Comment by Manzito13** [comment]
> 
> 
> Despite the GUI showing those k=25 plots as farming, they will _never_ win a reward on mainnet. Anything less than k=32 will be skipped over when proving.
wow .. ok .. I assume then that your answer clears all doubts about it .. thanks ... maybe someone should tell the developers to make the pertinent modifications so this no longer leads to confusion ...

**Comment by Manzito13** [comment]
> 
> 
> > Despite the GUI showing those k=25 plots as farming, they will _never_ win a reward on mainnet. Anything less than k=32 will be skipped over when proving.
> 
> That's what I thought - k25 is only for testnet and has no function on mainnet. @Manzito13 I suggest you delete those plots and begin plotting some k32s because you can't win anything with a k25.

ok .. thanks .. All doubts cleared then !!

**Comment by Manzito13** [comment]
Amazing !! .. it even tells me the estimated time to win ... haha ... anyway! ... 
![Incredible2](https://user-images.githubusercontent.com/85766978/121793864-939fd780-cbd9-11eb-82d4-a257ef4717de.jpg)

**Comment by tjoker911** [comment]
How does on plot k33? I don't see a parameter option for it?

**Comment by VertiHydro** [comment]
> How does one plot k33? I don't see a parameter option for it?

You can't, not yet: #25 , #65
